runbooks/runbook.md

```diff
@@ -0,0 +1,120 @@
+# UPP - Methode Article Mapper
+
+Methode Article Mapper transforms native Methode content into content according to UPP json format. It is part of the so called Old Publishing Pipeline in UPP.
+
+## Primary URL
+
+https://upp-prod-delivery-glb.upp.ft.com/__methode-article-mapper/
+
+## Service Tier
+
+Platinum
+
+## Lifecycle Stage
+
+Production
+
+## Delivered By
+
+content
+
+## Supported By
+
+content
+
+## Known About By
+
+- hristo.georgiev
+- robert.marinov
+- elina.kaneva
+- georgi.ivanov
+- tsvetan.dimitrov
+- kalin.arsov
+- mihail.mihaylov
+- boyko.boykov
+- donislav.belev
+- dimitar.terziev
+
+## Host Platform
+
+AWS
+
+## Architecture
+
+Methode Article Mapper is a Dropwizard application which receives native Methode content (either from a Kafka topic or HTTP) and maps articles into content according to UPP JSON format.
+
+## Contains Personal Data
+
+No
+
+## Contains Sensitive Data
+
+No
+
+## Dependencies
+
+- kafka-proxy
+- document-store-api
+- public-concordances-api
+
+## Failover Architecture Type
+
+ActiveActive
+
+## Failover Process Type
+
+FullyAutomated
+
+## Failback Process Type
+
+FullyAutomated
+
+## Failover Details
+
+The service is deployed in both Delivery clusters. The failover guide for the cluster is located here:
+https://github.com/Financial-Times/upp-docs/tree/master/failover-guides/delivery-cluster
+
+## Data Recovery Process Type
+
+NotApplicable
+
+## Data Recovery Details
+
+The service does not store data, so it does not require any data recovery steps.
+
+## Release Process Type
+
+PartiallyAutomated
+
+## Rollback Process Type
+
+Manual
+
+## Release Details
+
+Manual failover is **not required** when a new version of the service is deployed to production. Releasing of no mapper *should not trigger* an automated failover. But if in doubt, do a faiolver.
```

Comment:
"Releasing of no mapper"?

The information here is not correct. Unfortunately the "methode-article-mapper" is coupled with the "read" healthcheck category so a release albeit short in time, may cause a failover.

Reply:
Thanks for the sharp eye on this, @karsov The failover instructions are altered.